src/osx/textctrl_osx.cpp

```diff
@@ -89,8 +89,10 @@ bool wxTextCtrl::Create( wxWindow *parent,
     DontCreatePeer();
     m_editable = true ;
 
+#if !defined(__WXOSX_COCOA__)
```

Comment:
This doesn't make sense to me, if `wxTextCtrl` without a border can work, we should just remove this.

Reply:
There is only iOS implementation now, but it's probably completely different border-wise anyhow.